lambda/src/search-request-logger/search-request-logger.mjs

```diff
@@ -1,6 +1,25 @@
 import { get } from 'lodash/object'
 import fetch from 'node-fetch'
 
+const getAuthorizationHeaderFromMessage = messageUser => {
+  if (!messageUser) {
+    return 'unauthenticated'
+  }
+
+  if (messageUser.idToken) {
+    return messageUser.idToken
+  }
```

Comment:
We can remove this. There's no case now where we'll pass an idToken value in the SNS message.